android/src/main/java/com/learnium/RNDeviceInfo/RNDeviceModule.java

```diff
@@ -349,6 +349,31 @@ public String getCarrierSync() {
   @ReactMethod
   public void getCarrier(Promise p) { p.resolve(getCarrierSync()); }
 
+  @ReactMethod(isBlockingSynchronousMethod = true)
+  public List<String> getCarriersSync() {
+    List<String> carriersNames = new ArrayList<>();
+    if (telMgr != null) {
+       if (android.os.Build.VERSION.SDK_INT >= android.os.Build.VERSION_CODES.LOLLIPOP_MR1){
+        SubscriptionManager subscriptionManager = (SubscriptionManager) getSystemService(Context.TELEPHONY_SUBSCRIPTION_SERVICE);
+        List<SubscriptionInfo> subscriptionInfos = subscriptionManager.getActiveSubscriptionInfoList();
+        for (int i = 0; i < subscriptionInfos.size(); i++) {
+            carriersNames.add(subscriptionInfos.get(i).getCarrierName().toString());
+        }
+       }
+       else{
```

Comment:
not just the leading space for indents, but spaces between control flow keywords and new braces please
```suggestion
       else {
```